src/main/java/com/toofifty/easyblastfurnace/methods/MetalBarMethod.java

```diff
@@ -58,9 +58,16 @@ public MethodStep next(BlastFurnaceState state)
             return waitForBars;
         }
 
```

Comment:
This can be rewritten as:

```JAVA
        if (state.getConfig().tickPerfectMethod() && state.getFurnace().hasEnoughBars(barItem()) ||
            !state.getConfig().tickPerfectMethod() && state.getFurnace().has(barItem())
        ) {
            return collectBars;
        }
```